src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -11,7 +11,22 @@ method java.util.Collection size
 method java.util.Map put java.lang.Object java.lang.Object
 method java.util.List get int
 new java.net.URL java.lang.String
+method java.lang.String contains java.lang.CharSequence
+method java.lang.String endsWith java.lang.String
+method java.lang.String indexOf java.lang.String
+method java.lang.String toLowerCase
+method java.lang.String matches java.lang.String
 method java.lang.String split java.lang.String
+method java.lang.String startsWith java.lang.String
+method java.lang.String replace java.lang.CharSequence java.lang.CharSequence
+method java.util.Collection contains java.lang.Object
+staticMethod java.lang.System getenv java.lang.String
```

Comment:
I found it in couple of our combination filters so I am interested in this one. It i imho impossible to keep the env-vars private if we allow master to run builds (even matrix parents). Perhaps it is not even necessary.